webview/src/stories/Table.stories.tsx

```diff
@@ -46,7 +47,8 @@ export default {
     design: {
       type: 'figma',
       url: 'https://www.figma.com/file/AuQXbrFj60xA2QXOjo9Z65/Experiments-Panel-%E2%80%A2-496'
-    }
+    },
+    layout: 'fullscreen'
```

Comment:
This allows the table stories to work with the `position:fixed`, as Storybook adds a padding to its `iframe` by default which is considered part of the viewport which messes with the table otherwise.